src/material/menu/menu.ts

```diff
@@ -552,6 +552,8 @@ export class _MatMenuBase
     '[attr.aria-labelledby]': 'null',
     '[attr.aria-describedby]': 'null',
     'ngSkipHydration': '',
+    // This binding is used to ensure that the component ID doesn't clash with the `MatTextColumn`.
```

Comment:
question: Is the Menu component at risk of clashing with `MatTextColumn` or is this a copy-paste error?

Reply:
It's just me copying the same blob everywhere.

Reply:
Which one is the correct one? `MatLegacyMenu`?

Reply:
Resolved in 320a3b8c2e469fc360ed606a031df6b863e6f699.